api/src/utils/github.js

```diff
@@ -58,11 +58,33 @@ export const getUserByName = async (username) => {
   log.debug(`getUserByName ${username}`)
   const installations = await getAuthenticatedApps()
   const config = getConfig()
-
   const response = await installations.apps[
-    config.primaryOrg
+    config.primaryOrg.toLowerCase()
   ].authenticatedRequest('GET /users/{username}', {
     username,
   })
   return response.data
 }
+
+export const getRequestStatuses = async (requests) => {
+  log.info(`getRequestStatuses`)
+  let pendingUserRequests = []
+  const installations = await getAuthenticatedApps()
+
+  const config = getConfig()
+
+  for (const org of config.orgs) {
+    log.debug(`org being searched ${org}`)
+    const orgPendingRequests = await installations.apps[
+      org.toLowerCase()
+    ].authenticatedRequest('GET /orgs/{org}/invitations', {
+      org,
+    })
+    pendingUserRequests = pendingUserRequests.concat(Object.values(orgPendingRequests.data).filter
+    (match => requests.map((requestInDB) => {
+      requestInDB.recipient === match.login
+    })))
```

Comment:
this routine uses 3 difference array functions (3 loops) inside of this outer for loop. I think there is probably a better way to represent what you are trying to do